spec/active_force/active_query_spec.rb

```diff
@@ -188,77 +191,118 @@
     end
     let(:api_result1) do
       [
-        {"Id" => "0000000000AAAAABBB"},
-        {"Id" => "0000000000CCCCCDDD"},
-        {"Id" => "0000000000EEEEEFFF"}
+        { 'Id' => '0000000000AAAAABBB' },
+        { 'Id' => '0000000000CCCCCDDD' },
+        { 'Id' => '0000000000EEEEEFFF' }
       ]
     end
     let(:api_result2) do
       [
-        {"Id" => "0000000000EEEEEFFF"}
+        { 'Id' => '0000000000EEEEEFFF' }
       ]
     end
     it 'allows method chaining' do
-      result = active_query.where("Text_Label = 'foo'").where("Checkbox_Label = true")
+      result = active_query.where("Text_Label = 'foo'").where('Checkbox_Label = true')
       expect(result).to be_a described_class
     end
 
     context 'when calling `where` on an ActiveQuery object that already has records' do
       it 'returns a new ActiveQuery object' do
         first_active_query = active_query.where("Text_Label = 'foo'")
         first_active_query.inspect # so the query is executed
-        second_active_query = first_active_query.where("Checkbox_Label = true")
+        second_active_query = first_active_query.where('Checkbox_Label = true')
         second_active_query.inspect
         expect(second_active_query).to be_a described_class
         expect(second_active_query).not_to eq first_active_query
       end
     end
-
   end
 
-  describe "#find_by" do
-    it "should query the client, with the SFDC field names and correctly enclosed values" do
+  describe '#find_by' do
+    it 'should query the client, with the SFDC field names and correctly enclosed values' do
       expect(client).to receive :query
       active_query.find_by field: 123
-      expect(active_query.to_s).to eq "SELECT Id FROM table_name WHERE (Field__c = 123) LIMIT 1"
+      expect(active_query.to_s).to eq 'SELECT Id FROM table_name WHERE (Field__c = 123) LIMIT 1'
     end
   end
 
-  describe "#find_by!" do
-    it "raises if record not found" do
+  describe '#find_by!' do
+    it 'raises if record not found' do
       allow(client).to receive(:query).and_return(build_restforce_collection)
-      expect { active_query.find_by!(field: 123) }.to raise_error(ActiveForce::RecordNotFound)
+      expect { active_query.find_by!(field: 123) }
+        .to raise_error(ActiveForce::RecordNotFound, "Couldn't find #{sobject.table_name} with {:field=>123}")
     end
   end
 
-  describe "responding as an enumerable" do
+  describe '#find!' do
```

Comment:
Same on this file.  I changed the test above for `find_by!` to check the exception message and added the tests for `find!`.